presto-main/src/main/java/com/facebook/presto/operator/HashAggregationOperator.java

```diff
@@ -467,6 +467,8 @@ private void closeAggregationBuilder()
         if (aggregationBuilder != null) {
             aggregationBuilder.recordHashCollisions(hashCollisionsCounter);
             aggregationBuilder.close();
+            // aggregationBuilder.close() will release all memory reserved in memory accounting.
+            // The reference must be set to null afterwards to avoid unaccounted memory.
```

Comment:
We had a discussion with @haozhun and @dain and thought it would be dangerous if we accidentally remove the set reference to null line in the future. Hence the comment.